openid-caep-specification-1_0.txt

```diff
@@ -630,55 +630,64 @@ Cappalli & Tulshibagwale     Standards Track                   [Page 11]
 
 3.4.1.  Event-Specific Claims
 
-   current_level  REQUIRED, JSON string: the current NIST Authenticator
-      Assurance Level (AAL) as defined in [SP800-63R3]
+   namespace:  REQUIRED, JSON string: the namespace of the values in the
+      current_level and previous_level claims.  This string MAY be one
+      of the following strings:
 
-      This string MUST be one of the following strings:
+      *  RFC8176: The assurance level values are from the [RFC8176]
+         specification
 
-      *  nist-aal1
+      *  RFC6711: The assurance level values are from the [RFC6711]
+         specification
 
-      *  nist-aal2
+      *  ISO-IEC-29115: The assurance level values are from the
+         [ISO-IEC-29115] specification
 
-      *  nist-aal3
+      *  NIST-IAL: The assurance level values are from the
+         [NIST-IDPROOF] specification
 
-   previous_level  REQUIRED, JSON string: the previous NIST
-      Authenticator Assurance Level (AAL) as defined in [SP800-63R3]
+      *  NIST-AAL: The assurance level values are from the [NIST-AUTH]
+         specification
 
-      This MUST be one of the following strings:
+      *  NIST-FAL: The assurance level values are from the [NIST-FED]
+         specification
 
-      *  nist-aal1
+      *  Any other value that is an alias for a custom namespace agreed
+         between the Transmitter and the Receiver
 
-      *  nist-aal2
+   current_level  REQUIRED, JSON string: The current assurance level, as
```

Comment:
JSON is technically case sensitive. Should we say that this string is case-insensitive?